packages/amplify-e2e-tests/src/__tests__/delete.test.ts

```diff
@@ -171,60 +171,3 @@ async function timeout(timeout: number) {
     setTimeout(resolve, timeout);
   });
 }
-
```

Comment:
Was this deletion intentional?

Reply:
yeah it was removed in master, but I messed up the merge